Bottom: 9b09bb2b4cbc12c6d1f03168efe59c5366f16640
Top:    8e27942fc70117bfc92d805e1ec88e6520335e9b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-21 21:25:56 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index 598324e..a4f9c3d 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -333,13 +333,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
